### ISSUE TITLE
Pro User: Weather has vanished

### ISSUE THREAD

**Issue by hobhobuk** [8bitPit/Niagara-Issues#1150]
Weather has vanished, unsure why.  Woke up and it was gone.  Tried restarting, still not there.  Just see battery percentage. 

- Samsung A80 (SM-A805F)
- Android V9

**Comment by 8bitPit** [comment]
I'm sorry that the widget isn't showing up. Does it still not work?

**Comment by hobhobuk** [comment]
Has reappeared now. Actually reappeared after a phone update came through
so maybe was something to do with Samsung UI?

**Comment by 8bitPit** [comment]
I don't know, but I'm glad it works again. I believe weather didn't show up either because Niagara Launcher hadn't access to the internet or because Niagara's server / Dark Sky's weather services were down. However I didn't receive other messages that weather wasn't accessible. 
Please reopen this issue on case vanishes again.

**Comment by YannisFraselle** [comment]
Hello I had similar thing today, weather has vanished. I tried to switch off the option in the parameters and then switch on several time, didn't work. I tried also to restart my phone, didn't work aswell. (I was connected to internet for information). And when I touch the option to change the location the pop up take 5 big sec to show up but my city wasn't in the option, I close and reopen the option and a this moment my city was back, I selected it and the weather on the screen was back.
